docs/source/quick-start.rst

```diff
@@ -0,0 +1,136 @@
+Quick Start
+===========
+
+Install NuPIC
+-------------
+
+.. code-block:: bash
+
+   pip install nupic [--user]
+
+The ``--user`` is in brackets because it is optional. See the
+`pip reference guide <https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-user>`_
+for details.
+
+The output of this command should end with:
+
+    Successfully installed nupic-X.X.X nupic.bindings-X.X.X
+
+If your installation was unsuccessful, you can find help on
```

Comment:
Consider adding "where X.X.X represents the most recent version shown at https://pypi.python.org/pypi/nupic"